cypress/e2e/tests/pages/explorer/more-resources/api/custom-resource-definitions.spec.ts

```diff
@@ -49,6 +50,8 @@ describe('CustomResourceDefinitions', { testIsolation: 'off', tags: ['@explorer'
     });
 
     it('pagination is visible and user is able to navigate through crd data', () => {
+      cy.tableRowsPerPage(10);
+      cy.reload(true);
```

Comment:
add `cy.tableRowsPerPage(10);` in before hook instead of the `it` block to avoid use of reload

Reply:
We should start accounting for idempotency and each test should set they own environmental settings/contrains. Not the entire suite.

Reply:
- the rows per page change is persisted as a `userpreferences` resource in the upstream cluster and will remain set until updated. the exception being if there's an intercept on `/v1/userpreference...`
- we try to avoid reload/<page>.goTo's as much as possible, tests balloon in duration when added on a per test basis which end up blocking gh/org runners

Reply:
I'll move these to the Setup section, I'll still have to reload there but is possible going to be just once per tested list (pods, users, etc) as originally suggested by @yonasberhe23